components/brave_wallet_ui/utils/address-utils.ts

```diff
@@ -14,3 +18,14 @@ export function isValidAddress (value: string, length: number): boolean {
 
   return true
 }
+
+export function isHardwareAccount (accounts: WalletAccountType[], address: string) {
+  for (const account of accounts) {
+    if (account.deviceId &&
+        // TODO(darkdh): https://github.com/brave/brave-browser/issues/18938
```

Comment:
This issue is closed, is this TODO still valid?

Reply:
It's no longer valid and please do not use this `toLowerCase` comparison here

Reply:
removed